lms/djangoapps/shoppingcart/migrations/0026_migrate_invoices.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+from decimal import Decimal, ROUND_DOWN
+from south.v2 import DataMigration
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """
+        Write your forwards methods here.
+        """
+        # Note: Remember to use orm['appname.ModelName'] rather than "from appname.models..."
```

Comment:
Please remove the auto-generated comments from south (both this comment and the docstring)